backend/routes/stocks.js

```diff
@@ -56,37 +57,23 @@ router.get("/stocks/:ticker", async (req, res) => {
             return;
         }
 
-        const user = req.session.user;
+        const StockId = stock.id;
+        const UserId = user.id;
 
-        if (user != null) {
```

Comment:
Why is this check not necessary?

Reply:
Users have to be logged in now to use the app and the checkSession middleware makes sure a user exists.